openapi-spec/swagger.yaml

```diff
@@ -87,7 +92,104 @@ tags:
     externalDocs:
       description: Find out more
       url: 'https://sodafoundation.io/'
+# schemes:
+# - http
 paths:
+  '/v1/{project_id}/aksks':
+    post:
+      tags:
+        - AKSK
+      summary: Create a new AKSK for a User.
+      operationId: addAKSK
+      parameters:
+        - name: project_id
+          in: path
+          required: true
+          description: AKSK to be generated for User of a Project / Tenant. project_id is same as tenant_id.
+          schema:
+            type: string
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AKSK'
```

Comment:
POSt api response will have project_id and user_id? I thought it will be AK SK generated 
![image](https://user-images.githubusercontent.com/48085413/123683441-d2848d00-d869-11eb-8e74-d922aaf05017.png)

Reply:
Fixed

Reply:
![image](https://user-images.githubusercontent.com/65751939/123685512-6b1c0c80-d86c-11eb-8a52-8792bdd2ceca.png)